app-backend/db/src/main/scala/com/azavea/rf/database/filters/Filterables.scala

```diff
@@ -1,267 +1,373 @@
 package com.azavea.rf.database.filter
 
-import java.sql.Timestamp
 import java.util.UUID
 
-import com.azavea.rf.database.meta.RFMeta
+import cats.implicits._
 import com.azavea.rf.database.Filterable
+import com.azavea.rf.database.meta.RFMeta
 import com.azavea.rf.datamodel._
-import cats.implicits._
+import com.typesafe.scalalogging.LazyLogging
+import doobie.Fragments.in
 import doobie._
 import doobie.implicits._
-import doobie.Fragments.in
 import doobie.postgres._
 import doobie.postgres.implicits._
-import cats._
-import cats.data._
-import cats.effect.IO
 import geotrellis.vector._
 
-import com.typesafe.scalalogging.LazyLogging
-
+@SuppressWarnings(Array("CatchException", "ListAppend"))
```

Comment:
I suppressed these at the `trait` level because I was unable to set the annotations at the `implicit val` level.